code/pro-manager/src/main/java/com/cs673olsum24/promanager/serviceImpl/TaskServiceImplementation.java

```diff
@@ -93,6 +93,15 @@ public Map<String, Object> addTasks(HttpServletRequest request, Map<String, Obje
 		return map;
 	}
 
+	public Map<String, Object> deleteTask(String id) {
+        Map<String, Object> map = new HashMap<>();
+
+        this.taskDAO.deleteTask(id);
+
+        map.put("Response", "Success");
```

Comment:
Move this.taskDAO.deleteTask(id) to a try exception. In exception, return Response as Failure